pinot-core/src/main/java/org/apache/pinot/core/indexsegment/generator/SegmentGeneratorConfig.java

```diff
@@ -595,12 +595,12 @@ public void setOnHeap(boolean onHeap) {
     _onHeap = onHeap;
   }
 
-  public boolean isCheckTimeColumnValidityDuringGeneration() {
-    return _checkTimeColumnValidityDuringGeneration;
+  public boolean isSkipTimeValueCheck() {
```

Comment:
This is a backward-incompatible change , please flag the PR in the commit message. Will be good if you can identify the checkin/date when the config was introduced, so that we can use that during pinot release.

Reply:
Good suggestion. Added to the PR description